oasislmf/pytools/gul/core.py

```diff
@@ -116,6 +116,33 @@ def split_tiv(gulitems, tiv):
             gulitems[j] *= f
 
 
+@njit(cache=True, fastmath=True)
+def split_tiv_multiplicative(gulitems, tiv):
+    """Split the total insured value (TIV) using a multiplicative formula for the
+    total loss as tiv * (1 - (1-A)*(1-B)*(1-C)...), where A, B, C are damage ratios
+    computed as the ratio between a sub-peril loss and the tiv. Sub-peril losses
+    in gulitems are always back-allocated proportionally to the losses.
+
+    Args:
+        gulitems (numpy.array[oasis_float]): array containing losses of all items.
+        tiv (oasis_float): total insured value.
+    """
+    Ngulitems = gulitems.shape[0]
+    total_loss = 1.
```

Comment:
I would call that insure_value_left or undamaged_value or 1_minus_total_loss.
I prefer to avoid confusion when reading as until line 135 it is not the total loss

Reply:
changed `total_loss` to `undamaged_value`